src/main/java/frc/robot/subsystems/Input.java

```diff
@@ -15,6 +16,7 @@ private Input() {
 
   private static final Joystick mRotJoystick = new Joystick(JoystickConstants.ROT_JOYSTICK_PORT);
   private static final Joystick mTransJoystick = new Joystick(JoystickConstants.TRANS_JOY_PORT);
+  private static final XboxController mXBOX = new XboxController(JoystickConstants.XBOX_CONTROLLER_PORT);
```

Comment:
This should be mXbox or mXboxController(preferably the second one since that's the one we used last year)